packages/spor-react/src/pagination/Pagination.tsx

```diff
@@ -1,143 +1,207 @@
-        // If selectedPage is near the end, display the last pages.
-        displayPageNumbers.push(1);
-        displayPageNumbers.push("...");
-        displayPageNumbers.push(
-          ...Array.from(
-            { length: maxVisiblePages - 1 },
-            (_, i) => totalPages - maxVisiblePages + 2 + i,
-          ),
-        );
-      } else {
-        // Display pages with "..." in the middle.
-        displayPageNumbers.push(1);
-        displayPageNumbers.push("...");
-        for (
-          let i = selectedPage - Math.floor((maxVisiblePages - 3) / 2);
-          i <= selectedPage + Math.floor((maxVisiblePages - 3) / 2);
-          i++
-        ) {
-          displayPageNumbers.push(i);
-        }
-        displayPageNumbers.push("...");
-        displayPageNumbers.push(totalPages);
-      }
-    }
-    return displayPageNumbers.map((pageNumber, index) =>
-      pageNumber === "..." ? (
-        <Box as="li" key={index} css={styles.listItem}>
-          <Center>...</Center>
-        </Box>
-      ) : (
-        <Link
-          key={index}
-          as={Box}
-          onClick={() => {
-            if (pageNumber !== "...") {
-              onPageChange(+pageNumber);
-            }
-          }}
-          padding={pageNumber === "..." ? 0 : undefined}
-          css={pageNumber === selectedPage ? styles.activeButton : styles.link}
+  return (
+    <ListItem>
+      <ChakraPagination.Item asChild ref={ref} {...props}>
+        <Box
+          css={props.value === page ? styles.activeButton : styles.listItem}
+          aria-label={`${t(texts.page)} ${props.value} ${t(texts.of)} ${totalPages}`}
+          as={props.as || "a"}
         >
-          {pageNumber}
-        </Link>
-      ),
-    );
-  };
+          {props.value}
+        </Box>
+      </ChakraPagination.Item>
+    </ListItem>
+  );
+});
+export const PaginationPrevTrigger = React.forwardRef<
+  HTMLButtonElement,
+  ChakraPagination.PrevTriggerProps
+>(function PaginationPrevTrigger(props, ref) {
+  const { page } = usePaginationContext();
+  const recipe = useSlotRecipe({ key: "pagination" });
+  const styles = recipe();
+  const { t } = useTranslation();
 
+  if (page <= 1) {
+    return null;
+  }
   return (
-    <Flex as="nav" aria-label="pagination">
-      <Box
-        as="ul"
-        display="flex"
-        listStyleType="none"
-        gap={[0, 1]}
-        padding={0}
-        margin={0}
-      >
-        <Box as="li" aria-label={t(texts.previousPage)}>
-          <TextLink
-            onClick={() => onPageChange(selectedPage - 1)}
-            css={hasPreviousPage ? styles.link : styles.disabled}
-          >
-            <DropdownLeftFill18Icon css={styles.icon} />
-          </TextLink>
+    <ListItem>
+      <ChakraPagination.PrevTrigger ref={ref} asChild {...props}>
+        <Box
+          css={styles.listItem}
+          aria-label={t(texts.previousPage)}
+          as={props.as || "a"}
+        >
```

Comment:
Jeg opplever et problem her. Siden <a> som er rendret har ikke en href attribute og handling er håndtert av javascript, da er navigasjon med tastatur ikke synlig <a> må ha href for å være focusable, kan du prøve å gi en rolle som button `<a role="button">` for å se om det blir gjenkjent? alternative er at previous og next blir as="button"